### ISSUE TITLE
Better documentation for the Maps module of the library

### ISSUE THREAD

**Issue by adnanhemani** [data-8/datascience#387]
Once the new version of the Maps module is released, we should add better documentation to it so it's more user-friendly.

**Comment by AdilGM** [comment]
Hey Adnan, has the new version released? I can help add documentation if needed.

**Comment by adnanhemani** [comment]
@AdilGM yes. the current version of the library is the latest (and stable version). Please feel free to start with the documentation if you'd like!
